НерфСтанов

Jordanstolethepreciousslam opened the pull request:
## Описание
Основные изменения касаются нелетального оружия в целом, и дальнего боя в частности. Для большинства нелетального оружия в игре время оглушения было урезано до минимальных значений, а также добавлен накапливающийся урон выносливости. Основным вектором изменений является продление и усложнение нелетального боя. На практике это означает что вы столкнетесь с тем что нелетальное оружие на первый взгляд стало совершенно непригодным для боя и задержания. Тазеры больше не оглушают а лишь разоружают, дубинки оглушают на мизерное время и наручники надеваются неприлично долго. Чтобы избежать недопониманий я сразу укажу как подразумевается игра при новой боевой системе.

1. Для большинства нелетального оружия теперь действует правило - УДАРИТЬ ОДИН РАЗ НЕДОСТАТОЧНО. Ударив один раз вы лишь уроните противника на пару секунд. Чтобы захватить жертву вам нужно попасть по противнику НЕСКОЛЬКО раз подряд. Для обычного человека без наркотиков требуется 4-5 ударов станбатоном, либо 4 попадания тазером или дисейблером.
Для станбатонов СБ это тривиальная задача, так как жертва всё равно не сможет встать между ударами, однако самодельная дубинка единичным попаданием лишь разоружает как тазеры, но не делает жертву сразу беспомощной.

2. Тазеры теперь не является основным оружием, а high risk - high reward альтернативой парализатору(дисейблеру). Подразумевается что малая скорострельность тазера компенсируется эффектом разоружения при попадании. Дисейблер стреляет быстро, но тазер обезоруживает жертву заставляя оппонента тратить время на подъем своего оружия и прицеливание. Попробуйте комбинировать и менять на ходу эти режимы для наилучшего эффекта.

3. Нет это не нерф СБ. Изменению подверглись также и большинство антагонистов и их снаряжения. Полный список изменений вы можете увидеть на гитхабе или в нашем дискорде. Также приветствуются отзывы, репорт багов и указание на недочёты баланса и возникающие сложности.

## Описание

Этот патч ставит однокнопочные станы туда, где им самое место.

-Большинству источников станов в игре урезано время стана до минимума и добавлен стамина урон. Теперь захват живьем можно совершить только либо через граб, либо через дисейблер, либо через продолжительный массаж станами

-Это изменение нивелирует станлоки как явление, за исключением стан оружия ближнего боя, которое всё еще может станлочить несмотря на то что стана не хватит на наручники)
-Также подверглись изменению наручники(надеваются дольше на 50%), флеш гранаты и часть наркотиков(дают больше регена стамины).
Изменению также подверглась подавляющая часть источников стана у антагонистов. Энерго арбалет, снайперские пули, mind batterer, Заклинания вампиров и тенеморфов, культа и снаряжение мага,

Кроме того усилен бакшот и ослаблены резиновые пули и дробь для дробовика, также ослаблены слаги.
Уменьшено время случайных станов в ближнем бою для безоружного и не очень боя.
Уменьшен стан с большинства источников подскальзывания.
Увеличен урон самодельного арбалета для соответствия урону самодельной пневмопушке.
ЦНС ребутер теперь восстанавливает стамину если её суммарный урон выше 70.

ЧаВо
Q: А ты же в курсе что тазеры ИРЛ...
A: У нас не тазеры ИРЛ, хватит, пожалуйста.

Q: А ты мыло понерфил?
A: Да. Полный список понерфленных и переработанных вещей можно найти в чейнджлогах которые я кидаю.

Q: Ну и как теперь за СБ играть!!7!? Кроме летала ничего не работает!11
A: Просто стреляй/бей несколько раз. Если после 8-10 ударов/попаданий жертва всё еще быстро встает значит она под наркотой/имплантами/абилками(либо вы пытаетесь задержать человека телескопичкой, что без граба теперь невозможно).

Q: Как теперь за антагов играть!77!111? Цель кричать успевает11!!1!
A: Позвать на помощь могли и в старой системе станов. Если вам удавалось украсть кого то беззвучно значит вам попался нерасторопный игрок. Конечно сейчас таким игрокам даются дополнительные шансы, но игрок распечатывает призыв о помощи в радиоканал вместо попытки бегством - считайте что вам повезло.

Jordanstolethepreciousslam commented:
Ах да, еще понерфлены Ксеноморфы. Их дизарм теперь станит с 40% шансом на 1 тик вместо 100% шанса на 5 тиков и наносит 20 стамина урона.

Sykybekk commented:
Это просто ахуенно

Jordanstolethepreciousslam commented:
Чуть конкретнее
-Тазеры, энергоарбалеты и иже с ними станят от 0.1 до 1 секунды  вместо прошлых 9 и наносят 15 стамина урона(30 у энерго арбалета)  
Мили(станпалки) станят на 2 цикла( от двух секунд до трех) и наносят 10 стамина урона. Исключением является только телескопичка с 80% шансом станящая на один цикл(от 0.1 секунд до 1), и с 20% на два.
Глейры станят на 3 цикла(около 4.5 секунд)
Вопли оглушают на 1-2 цикла
Случайные станы с мили ударов не стан оружия урезаны вполовину. с 4 циклов( 6 секунд) до 2( 3 секунды).
Флешбенги урезаны примерно наполовину.
подскальзывания урезаны наполовину (с 4 до 2 циклов)

Суть изменений. Больше нельзя захватить цель живым в два клика(один тазером, второй наручниками)
Либо заламывать цель грабом оглушая дубинкой, либо тратить время и заряд оружия накапливая стамина урон на цели(либо рискнуть и стрелять вдвойне эффективным по стамине, но не станящим дисейблером), либо использовать резиновые патроны. Либо стрелять леталом (щиткур mode/on).
Короче летал теперь действительно простой выбор а захват живьем требует определенной сноровки.

Вкратце - Всем вечный адренал бесплатно и навсегда, и пусть никто не уйдет обиженным

tizer7 commented:
жестковато однако. фактически, то что раньше называли станом - данный ПР умножает на 0. хотя мне нравится - больше горящих жоп щиткуров.
у ксено случайный диапазон стамина урона от 10 до 20 за удар, с шансом стамина урона 100%, я правильно понял?

PhantornRU commented:
Это охуенно. Дайте два!

kelgaras commented:
выглядит как буквально новый гемплей, заебись

AttHawk commented:
дизеблеру бы поднять стаминаурон до 35, а так выглядит страшно, но интересно, будем посмотреть примут ли это и как оно будет играться

HMBGERDO commented:
Вампир? Пошёл нахуй, вампир, ты теперь ничего не можешь.
СБ это не сильно изменит, они также будут сильнее всех из-за дизейблеров.
Тазеры не будут иметь смысла вообще, потому что 40 стамина урона против 10, что лучше? 
Станбатоны больше не имеют смысла, потому что цель встанет между ударами и убежит, соси хуй. 
Связать кого-то? Теперь это пиздец задачка, удачи хлопчик.

Короче если добавлять, то только в тестовом режиме на короткий период времени не на мейн сервер.

HMBGERDO commented:
а, и еще, карп теперь самый сильный в игре, кстати, с вот этим ПРом

rebel228 commented:
Мнение руководства сервера можно услышать по поводу этого ПРа, или уже всё решено и просто узнается мнение коммьюнити?

Bizzonium commented:
@rebel228 ничего ещё не решено. Опробовать можно тут [byond://play.ss220.space:7720](byond://play.ss220.space:7720)

Jordanstolethepreciousslam commented:
> Вампир? Пошёл нахуй, вампир, ты теперь ничего не можешь.
> СБ это не сильно изменит, они также будут сильнее всех из-за дизейблеров.
> Тазеры не будут иметь смысла вообще, потому что 40 стамина урона против 10, что лучше?
> Станбатоны больше не имеют смысла, потому что цель встанет между ударами и убежит, соси хуй.
> Связать кого-то? Теперь это пиздец задачка, удачи хлопчик.
> 
> Короче если добавлять, то только в тестовом режиме на короткий период времени не на мейн сервер.

Дисейблер кладет в стаминакрит с 3 выстрелов а 3 секунды недостаточно чтобы нанести второй удар станбатоном. Дальше не читал. Ты точно про наш билд?

Jordanstolethepreciousslam commented:
Текущие выявленные проблемы нуждающиеся в решении на 19.07.2021
-Глейр тенеморфов. Поднять время стана(и заодно время наручников) дабы можно было грабать чела но не связывать за время глейра. Вернуть старую немоту. Либо иное решение которое даст вампиру возможность пленить голыми руками но не за пару кликов. Короче переработать.
-Глейр вампира. То же самое что и с глейром тенеморфом. Хотя и можно использовать бутылки, станпрод или перчатки, возможно стоит сделать возможным захват живьем с одними наручниками и голыми руками. Короче переработать.
-Станы культяпок нуждаются в переработке
-Энергопотребление киборга
-снизить урон дизейблера с 30 до 20. У него выше скорострельность, так что пускай кладет в стаминакрит за 5 попаданий. вместо 4.
-Урезать станы со взломанного дефиба
-Урезать немного стан бипски 
-Проверить код на баг с вечным станом в грабе.
-Чутка твикнуть химикаты на стамина урон.

Commit:
Допиливание новой боевки.

-Глейры теперь роняют на достаточное время для связывания, но любые простые общедоступные химикаты уменьшающие стан имеют шанс сократить время стана до безопасных значений(никотин, эпинефрин и прочие)
-Тазеры теперь наносят 25 стамина урона(4 попадания до стаминакрита)
-Дисйблеры наносят 33 стамина урона(те же 4 попадания до стамина крита)
=разные значения урона(при одинаковых количествах попаданий до стаминакрита) будут иначе воздействовать на цели под наркотой(регенерирующей выносливость)
-Все апгрейды грабов ускорены на 10%
-Станы от дефибов порезаны
-Станбатоны наносят на 50% больше стамина урона (стамина крит за 7 ударов
-Мут от глейра шэдоулинга возвращен к старым значениям.
-Потребление энергии дисейблером борга уменьшено с 250 до 175
-Изменены значения восстановления стамины у некоторых наркотиков и химикатов

OpsCoretango commented:
у нас есть система боли?
т. е от сильной боли чел валится или не может двигатся.

Jordanstolethepreciousslam commented:
-Тазеры теперь наносят 25 стамина урона(4 попадания до стаминакрита)
-Дисйблеры наносят 33 стамина урона(те же 4 попадания до стамина крита)
-Станбатоны наносят на 50% больше стамина урона (стамина крит за 7 ударов)
-Повышен шанс стана телескопички на 2 тика вместо одного (с 20% до 50%)
=разные значения урона(при одинаковых количествах попаданий до стаминакрита) будут иначе воздействовать на цели под наркотой(регенерирующей выносливость)
-Все апгрейды грабов ускорены на 10%
-Станы от дефибов порезаны
-Станбатоны наносят на 50% больше стамина урона (стамина крит за 7 ударов)
-Мут от глейра шэдоулинга возвращен к старым значениям.
-Потребление энергии дисейблером борга уменьшено с 250 до 175
-Изменены значения восстановления стамины у некоторых наркотиков и химикатов
-Бипски понерфлен. Роняет на полсекунды и дает стаминакрит за 3 удара. Sleep tight comrade.
-Глейры теперь роняют на достаточное время для связывания, но любые простые общедоступные химикаты уменьшающие стан имеют шанс сократить время стана до безопасных значений(никотин, эпинефрин и прочие)
-Твикнуты следущие химикаты: Кровь Земли, Аранесп, Стимуляторный агент(адренал тритора), метамфетамин, эфедрин, эпинефрин

Jordanstolethepreciousslam committed:
Update vampire_powers.dm

Jordanstolethepreciousslam committed:
Update bullets.dm

Jordanstolethepreciousslam committed:
Update energy.dm

Jordanstolethepreciousslam committed:
Update batons.dm

Jordanstolethepreciousslam committed:
Update stunbaton.dm

Jordanstolethepreciousslam committed:
Update secbot.dm

Jordanstolethepreciousslam committed:
Update secbot.dm

Bizzonium commented:
Обновил тестовый сервер, после рестарта применятся изменения

Jordanstolethepreciousslam committed:
Переработка глейра тенелинга

Jordanstolethepreciousslam committed:
Переработка абилок вампира

Jordanstolethepreciousslam committed:
Починка переработки глейра тенелинга

pepebrain

Jordanstolethepreciousslam commented:
-Переработаны глейры вампира и тенеморфа. Они теперь станят на 3 секунды, но накладывают эффект который уложит цель в стамина крит через 5-6 секунд. Самое то чтобы подергатся перед смертью.
-Уменьшена цена в крови для следущих абилок вампира.
--Теневой шаг(30-->15)
--Заражение(100-->50)
--Летучие мыши(75-->50)

Jordanstolethepreciousslam commented:
-Увеличен стамина урон от трейторского арбалета(30-->55) позволяя стамина крит с 2 попаданий

Jordanstolethepreciousslam commented:
В поисках решения для балансировки карпа-наркомана я совсем забыл о том что я - непризнанный гений уровня Кодзимы.
Совсем недавно я бафал самодельные арбалеты и удавки. Самодельный арбалет и пневмопушка не стреляют прожектайлами, а значит игнорируют блок карпа. Такимм образом карп убивается копьями, стеклом и прочим метательным мусором.

Jordanstolethepreciousslam commented:
![image](https://user-images.githubusercontent.com/78968085/126507611-30ca8d9a-110b-4d77-ac35-50512336a845.png)

FlyingFloor commented:
Может я не прав, конечно, но чисто по тексту это очень сильно бафнет антагов. Инста-стан на 3 секунды, а потом ещё и стамина-крит. А чтобы уронить его, нужно ударить его 7 раз батоном, а ему нужно один раз бахнуть тебя глейром/два раза из арбалета. Как в задумке это должно баланситься? СБ/кекепаж должны на антага стаей накидываться чтобы у них хотя бы малейший шанс был? При условии, что по тем или иным причинам нельзя пострелять в антага, например. Или просто переходим к использованию летала до крита, а потом уже разбираться что антаг натворил? Реально интересно, как это в задумке должно все работать.
Кажется, что в этом билде станбатоны вообще не нужны будут.

Jordanstolethepreciousslam committed:
Твик граба

Jordanstolethepreciousslam committed:
Твик мимовского оружия

Jordanstolethepreciousslam committed:
Твик батона

Jordanstolethepreciousslam committed:
Твик режима казни доминатора

Jordanstolethepreciousslam committed:
Твик адренала

Jordanstolethepreciousslam committed:
Твик CNS Rebooter

Jordanstolethepreciousslam commented:
> Кажется, что в этом билде станбатоны вообще не нужны будут.

Батоны кладут в стан сразу. Отличие в том что этот стан слишком короткий для наручников но достаточен чтобы ударить снова. А вот стаминакрит уже достаточен для наручников

Jordanstolethepreciousslam committed:
Update stunbaton.dm

Jordanstolethepreciousslam committed:
Update augments_internal.dm

Jordanstolethepreciousslam committed:
Update energy.dm

Jordanstolethepreciousslam commented:
-Гипноз вампира теперь усыпляет цель становясь таким образом действительно осмысленной абилкой. Время применения увеличено.
-Триторский арбалет теперь станит как батон, но стаминакрит с 3 попаданий

Commit:
Этот патч ставит однокнопочные станы туда, где им самое место.

т.е у параши.
Большинству источников станов в игре урезано время стана до минимума и добавлен стамина урон. Теперь захват живьем можно совершить только либо через граб, либо через дисейблер, либо через продолжительный массаж станами(7 для тазеров, 10 для станбатонов)
Это изменение нивелирует станлоки как явление, за исключением стан оружия ближнего боя, которое всё еще может станлочить несмотря на то что стана не хватит на наручники)
Сильный нерф станпрода. Его стандартного зарядка не хватит для стамина крита. Только с использованием граба(или гарроты) можно захватить цель. Также подверглись изменению наручники(надеваются дольше на 50%) и часть наркотиков(дают больше регена стамины).
Кроме того усилен бакшот и ослаблены резиновые пули и дробь для дробовика.
Уменьшено время случайных станов в ближнем бою для безоружного и не очень боя.
Уменьшен стан с большинства источников подскальзывания.

Jordanstolethepreciousslam commented:
Ы

Ohar commented:
С принятием ЭТОГО задержание преступников на СБ превратится из «доминатор → наручники» в гачимучи с хватанием за попец и мацанием сосочков. Вы правда этого хотите?

Ну и СРП СБ точно придётся переписывать.

Jordanstolethepreciousslam commented:
> С принятием ЭТОГО задержание преступников на СБ превратится из «доминатор → наручники» в гачимучи с хватанием за попец и мацанием сосочков. Вы правда этого хотите?
> 
> Ну и СРП СБ точно придётся переписывать.

Как и должно быть. Задержание живьем сложнее убийства. А нелетальное оружие не должно быть в разы эффективнее летала. 

Переписывать СРП? Каво? У нас сейчас вполне прописаны ограничение на летал и они даже подразумевают что леталить проще чем убивать ТОЛЬКО ВОТ В НАШЕМ БИЛДЕ ЭТО НЕ ТАК)0)0)))).
Да им сейчас его надо переписать, запретив тазеры нахуй с поднятием кода чтобы противостояние не превращалось в ебаный цирк когда тритор использует крупнокалиберную револьвер будучи одетым в  элитный хардсьют синдиката а против него выходит челик с ТАЗЕРОМ и курткой на голую жопу и БУКВАЛЬНО ебет и сушит потом что ВООРУЖЕН ЛУЧШЕ чем тритор.

blind0 commented:
абсолютно точно ЗА, станы в текущем их виде слишком имбалансные и нелогичные.

Jordanstolethepreciousslam commented:
Полный чейнджлог на 02.11.2021
	Экипировка станции
		Нелетальное
			Дефибриляторы
				Оглушение при использовании уменьшено с 5 до 2
			Телескопический батон
				Оглушение уменьшено с 6 до 2 тиков
				Шанс в 25% при ударе оглушить на 3 тика.
			Светошумовые гранаты
				Максимальное оглушение уменьшено с 10 до 4 тиков.
			Наручники
				Время надевания увеличено с 3 до 5 секунд.
			Болы
				Увеличено время требуемое для снятия энерго-болы с 2 до 4 секунд.
				Увеличено время требуемое для снятия обычной болы с 3.5 до 6 секунд.
				Скорость полета бол утроена.
			Скользкие поверхности
				Оглушение от подскальзывания на скользких поверхностях снижено с 4 до 2 тиков.
			Станбатон
				Оглушение уменьшено с 7 тиков до 2.
				Теперь наносит 20 урона выносливости при ударе.
				Потребление заряда при ударе уменьшено с 1000 до 400.
			Станпрод
				Оглушение уменьшено с 5 до 2 тиков.
			Бипски
				Оглушение уменьшено с 5 до 2 тиков.
				Наносит 45 урона выносливости при ударе.
			Захваты
				Заполнение шкалы захвата теперь на 10% быстрее.
				Оглушение от удушающего захвата уменьшено с 5 до 2 тиков.
			ПДА клоуна
				Оглушение снижено с 8 до 5 тиков.
			Электрод
				Снижено потребление энергии этим снарядом для большинства оружия.
				Оглушение уменьшено с 5 до 1 тика.(Даже не стан а просто разоружение)
				Теперь наносит 25 урона выносливости при попадании.(4 до стамина крита)
			Парализатор
				Урон выносливости увеличен до 33.(4 до стамина крита)
			Оглушающий слаг
				Урон выносливости мнижен с 80 до 66.
			ТазерСлаг
				Оглушение уменьшено с 5  тиков до 1.
				Теперь наносит 25 урона выносливости.

	Летальное
		Случайное оглушение в ближнем бою для всех источников уменьшено с 4 тиков до 2(включает в себя как дизармы так и оглушения от двойного светового меча).
			Самодельный арбалет
				Урон повышен с 25 до 33.
				Добавлено 15 бронепробития. 
			LWAP
				Оглушение уменьшено с 5 до 2 тиков
				Теперь наносит 40 урона выносливости
			Режим казни Доминатора
				Урон увеличен с 40 до 50.
				Урон выносливости увеличен до 33.
			Картечь(buckshot)
				Урон увеличен с 12.5 до 14
				Бронепробитие увеличено с -30 до -20.
			Самодельная картечь
				Урон увеличен с 6 до 8
			Метеоритная пуля
				Оглушение уменьшено с 8 тиков до 2.

	Разное
		Никотин
			Шанс снижения стана уменьшен с 50% до 25%
			Восстановление выносливости удвоено.
		Метамфетамин
			Лечение оглушения уменьшено с 2.5 в тик до 2.
			Теперь восстанавливает 7 выносливости в тик.
		Аранесп
			Восстановление выносливости уменьшено с 40 до 9.
		Ультра смазка
			Восстановление выносливости увеличено с 2 до 7.
		Эфедрин
			Восстановление выносливости утроено.
		Кровь земли
			Восстановление выносливости снижено с 30 до 9.
		CNS Rebooter
			Оглушение при воздействии ЭМИ уменьшено с 5 тиков до 2.
			Более не снимает оглушение
			Теперь восстанавливает 9 выносливости в тик если суммарный урон выносливости превышает 60.

	Экипировка антагонистов
		Нелетальное
			Mind Batterer
				Разброс оглушения снижен с 4-7 тиков до 1-3.
				Теперь наносит случайный урон выносливости с разбросом 20-60.
			Ксеноморфы
				Шанс оглушения дизармом ксеноморфа уменьшен со 100% до 40%.
				Время оглушения уменьшено до 1 тика
				Теперь наносит от 10 до 20 урона выносливости при ударе.
			Пальцевый пистолет мима
				Оглушение уменьшено с 3 до 1 тика.
			Банановый посох
				Оглушение снижено с 5 тиков до 2.

		Летальное
			Энергетический топор
				Оглушение уменьшено с 4 до 3 тиков.
			Бензопила
				Оглушение уменьшено с 4 тиков до 1.
			Мьёльнир
				Оглушение уменьшено с 3 до 2 тиков
			Снайперская винтовка
				Оглушение уменьшено с 5 до 2 тиков.
			Плевок нейротоксином
				Оглушение снижено с 5 тиков до 1.
				Урон повышен с 5 до 33.
			Энергетический арбалет
				Оглушение снижено с 5 тиков до 2.
				Теперь наносит 40 урона выносливости.

		Способности
			Тенелинг
				Glare 
					Оглушение в упор уменьшено с 10 тиков до 2.
					Теперь наносит 70 урона выносливости
					Добавлен эффект DOT который наносит по 10 урона выносливости в течение 5 тиков
				Icy veins
					Оглушение уменьшено с 2 тиков до 1.
				Blindness smoke
					Оглушение уменьшено с 3 тиков до 2
			Вампир
				Glare
					Оглушение в упор уменьшено с 5 тиков до 2.
					Теперь наносит 70 урона выносливости
					Добавлен эффект DOT который наносит по 10 урона выносливости в течение 5 тиков
				Rejuvenate
					Мгновенное восстановление выносливости снижено с 75 до 60
					Теперь лечит по 10 урона выносливости в течение 5 тиков с момента применения.
				Hypnotize
					Время применения увеличено с 5 до 6 секунд.
					Более не оглушает
					Накладывает сон на 20 тиков.
				Deseased touch
					Стоимость в крови уменьшена со 100 до 50.
				Chiropteran Screech
					Оглушение уменьшено с 4 до 2 тиков.
					Наносит 60 урона выносливости.
				Summon Bats
					Стоимость в крови уменьшена с 75 до 50.
				Shadowstep
					Стоимость в крови уменьшена с 30 до 15.
			Предатель
				Адреналиновый имплант
					Восстановление выносливости увеличено в 5 раз.

Jordanstolethepreciousslam commented:
[Changelog.txt](https://github.com/ss220-space/Paradise/files/7460705/Changelog.txt)

Jordanstolethepreciousslam commented:
Читабельный вариант по ссылке

Jordanstolethepreciousslam commented:
Спорим без merge конфликтов не обойдется?

Gnatr commented:
ЦНС который регенит стамину, эми теперь обязательна будет что ли

azorko9695 commented:
Выглядит как что-то интересное и стоящее попытки
За
Причем делает геймплей интереснее и антагам и сбшникам

Jordanstolethepreciousslam committed:
Починка рофлокода и допиливание баланса абилок

Shadowstep - На этот раз цена снижена не только в названии. 30 ->20
Jaunt - Цена повышена для баланса нового эффекта гипноза и удешевления других абилок 30 -> 50
Summon Bats - Снижена реальная цена а не только название 75 -> 50
Deseased touch - Снижена реальная цена а не только название 100 -> 50

Jordanstolethepreciousslam committed:
Update stunbaton.dm

Твик батонов
Станбатон СБ - 2 тика стана и 25 стамины
Станпрод - 1тик стана(то есть просто разоружение) и 33 стамины.

Vallat commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -223,11 +222,15 @@
+		target.adjustStaminaLoss(70)
 		to_chat(target, "<span class='warning'>You are blinded by [user]'s glare.</span>")
 		add_attack_logs(user, target, "(Vampire) Glared at")
+		for(var/i = 1 to 5)
+			target.adjustStaminaLoss(10)
+			sleep(35)
```
Если у нас несколько целей, то с таким кодом стамина будет уменьшатся не у всех одновременно, а у каждого последовательно за другим. Т.е. каждый следующий человек будет ждать по 17.5 секунд пока его стамина не начнет уменьшаться. Решаем "асинхронностью"
```suggestion
		spawn for(var/i = 1 to 5)
			target.adjustStaminaLoss(10)
			sleep(35)
```

Vallat commented on code/modules/mob/living/carbon/human/human_defense.dm:
```diff
@@ -594,9 +594,11 @@ emp_act
-				apply_effect(5, WEAKEN, run_armor_check(affecting, "melee"))
-				add_attack_logs(M, src, "Alien tackled")
-				visible_message("<span class='danger'>[M] has tackled down [src]!</span>")
+				src.adjustStaminaLoss(rand(10,20))
+				if(prob(40))
+					apply_effect(1, WEAKEN, run_armor_check(affecting, "melee"))
+					add_attack_logs(M, src, "Alien tackled")
+					visible_message("<span class='danger'>[M] has tackled down [src]!</span>")
```
Попытку уронить надо тоже логировать, иначе в 60% случаев мы просто исключаем логирование взаимодействия между мобами

```suggestion
				src.adjustStaminaLoss(rand(10,20))
				if(prob(40))
					apply_effect(1, WEAKEN, run_armor_check(affecting, "melee"))
					add_attack_logs(M, src, "Alien tackled")
					visible_message("<span class='danger'>[M] has tackled down [src]!</span>")
				else
					visible_message("<span class='danger'>[M] tried to tackle down [src]!</span>")
					add_attack_logs(M, src, "Alien tried to tackle")
```

Vallat reviewed (changes requested):
На твики не смотрел, так как не занимаюсь балансом, но некоторые изменения механик надо доработать

Jordanstolethepreciousslam committed:
Update code/modules/mob/living/carbon/human/human_defense.dm

Логгирование попытки

Co-authored-by: Vallat <vox.crit2013@gmail.com>

Jordanstolethepreciousslam committed:
Update code/game/gamemodes/vampire/vampire_powers.dm

фикс метровой очереди за Стамина уроном kekwhat

Co-authored-by: Vallat <vox.crit2013@gmail.com>

Commit:
Замена Sleep() на полноценный статус эффект

Jordanstolethepreciousslam commented:
[Changelog.txt](https://github.com/ss220-space/Paradise/files/7532256/Changelog.txt)

SaingItsMe commented:
ахуенно пиздец как ахуенно оооооо заебись азиз бог и жордан тоже оооо

Commit:
Твики баланса
-Станпрод
--Урон выносливости уменьшен до 20(5 ударов до стамина-крита)
--Убрана пререзарядка станпрода из-за которой он оглушал в 3 раза реже задуманного
-Культ
--Оглушение
---Оглушение снижено с 5 тиков до 2
---Наносит 30 урона выносливости сразу
---Наносит еще 100 урона выносливости в течение 10 секунд после применения
-Предатель
--Mind Batterer
---Минимальный стамина урон повышен с 20 до 25

Commit:
Ребаланс БИ, генокрада и заклинаний культа

 CQC
-Цена повышена с 12 до 14
-Больше не может быть получен по скидке
-Шанс выхватывания оружия при дизарме снижен до 50%
    Комбинации
     Leg Sweep
         Длительность оглушения с ~5 до ~0.5 секунды
     ChokeHold
         Длительность сна уменьшена с ~30 до ~15 секунд
     Slam
         Длительность оглушения с ~10 до ~3 секунд
     Restrain
         Длительность оглушения с ~9 до ~3 секунд
     Kick
        Длительность сна с ~27 до ~5 секунд
        Урон мозгу с 15 до 4
Sleeping Carp
-Цена уменьшена с 17 до 16
-Оглушение с ударов на харме зависящее от здоровья цели уменьшено с ~7 до ~3 секунд
-Отключается если владелец подвержен любой зависимости (начиная с никотина и заканчивая омнизином).
     Wrist Wrench
        Длительность оглушения с ~5 до ~0.5 секунд
     Head Kick
        Длительность оглушения с ~7 до ~3 секунд
     Back Kick
        Длительность оглушения с ~7 до ~3 секунд
Заклинания культа
     Shadow Shackle
        Теперь не действует на цель если она не в стамина крите, обычном крите или спит.
     Stun
        Длительность немоты увеличена до ~9 секунд
Генокрад
     Tentacle
        Теперь дает пассивный граб вместо мгновенного агрессивного
     Epinephrine Overdose
        Теперь synaptizine восстанавливает по 5 выносливости в тик

Commit:
Замена stun на weaken у глейра тени

Потому что паралич это очень РПшно, но ничерта не очевидно оглушена цель или нет.

Jordanstolethepreciousslam commented:
[Changelog.txt](https://github.com/ss220-space/Paradise/files/7546279/Changelog.txt)

Commit:
Tweak: сила нелетального "огнестрела"

Стамина слаги
-Урон выносливости уменьшен с 66 до 55
Пули невидимого пистолета мима
-Урон выносливости уменьшен с 60 до 45
Пули трости мима
-Урон выносливости уменьшен с 60 до 45
Резиновая дробь
-Урон выносливости уменьшен с 20 до 15
Снаряды c-20r
-Урон выносливости уменьшен с 65 до 33
Экспансивные пули для стечкина
-Урон увеличен с 40 до 50

Jordanstolethepreciousslam committed:
Merge branch 'master220' into НерфСтанов

Jordanstolethepreciousslam committed:
Update uplink_item.dm

Jordanstolethepreciousslam commented:
[Changelog.txt](https://github.com/ss220-space/Paradise/files/7824919/Changelog.txt)

Commit:
Твик электрошока и переработка блика теней

Общие изменения
-длительность оглушения электрошока на гуманоидов уменьшена с 3 тиков до 2.
Блик тенелингов
 -Оглушение на расстоянии дальше 1 тайла заменено на обезоруживание и замедление в течение 5 тиков(~8 Секунд).
 -Немота на расстоянии дальше 1 тайла уменьшена вдвое(с 10 до 5 тиков).
Источники уменьшения оглушения
-Вероятность снижения оглушения сахаром уменьшена вдвое(с 50% до 25%)

Commit:
Бафф самых распространенных типов брони против энергетического оружия

Также уменьшает урон сферической пули в вакууме с 60 до 50(затрагивает револьверы и слаги)

Хардсьюты
				Инжинерный
					Повышено сопротивление энергетическому оружию с 5 до 15.
				Атмосферный
					Повышено сопротивление энергетическому оружию с 5 до 15.
				Элитный риг СЕ
					Повышено сопротивление энергетическому оружию с 5 до 25.
				Шахтерский
					Повышено сопротивление энергетическому оружию с 5 до 15.
				СБ
					Повышено сопротивление энергетическому оружию с 10 до 20.
				ГСБ
					Повышено сопротивление энергетическому оружию с 10 до 30.
			Спейссьют капитана
				Повышено сопротивление энергетическому оружию с 25 до 40.
			Бронежилет
				Повышено сопротивление энергетическому оружию с 10 до 20.
			Жилет СБ
				Повышено сопротивление энергетическому оружию с 5 до 15.
			Плащ ГСБ
				Повышено сопротивление энергетическому оружию с 10 до 25.
			Парадная броня капитана
				Повышено сопротивление энергетическому оружию с 10 до 20.
			Риот броня
				Повышено сопротивление энергетическому оружию с 10 до 20.
			Дюратканевый жилет
				Повышено сопротивление энергетическому оружию с 5 до 15.

Хардсьюты
				Кроваво-красный
					Повышено сопротивление энергетическому оружию с 15 до 30.
				Элитный
					Повышено сопротивление энергетическому оружию с 25 до 40.
				Экранированный(щитовой)
					Повышено сопротивление энергетическому оружию с 15 до 20.
			Спейссьют
				Повышено сопротивление энергетическому оружию с 15 до 25.
			Костюм Мага
				Повышено сопротивление энергетическому оружию с 20 до 30.

Хитин тенелинга
   Повышено сопротивление энергетическому оружию у врожденной брони с 0 до 10.

Jordanstolethepreciousslam commented:
[Uploading Changelog.txt…]()

Jordanstolethepreciousslam commented:
[Changelog.txt](https://github.com/ss220-space/Paradise/files/7864569/Changelog.txt)

Commit:
Твик парализатора и электрода, нерф омнизина, бафф арбалета

Омнизин
-Шанс развития зависимости увеличен в три раза
Электрод
-Урон выносливости повышен с 25 до 33
Дизейблер
-Урон выносливости понижен с 33 до 25
Энергоарбалет
-Цена снижена с 12 ТК до 9.

Commit:
Нерф космической смазки

Космическаская смазка
-Снижена длительность оглушения с 3 тиков до 2.

Длительность снижена в связи с тем что 3 тика - слишком долгое время оглушения в бою. Из-за легкости распыления смазки она часто используется антагонистами для отрыва от погони и преимущества в бою. Но так как погони стали сложнее для СБ(из-за нерфа тазеров) необходимость в долгом стане отсутствует. Также длительность в три тика позволяет настрелять упавшего до крита до того как он встанет. Этот нерф ставит смазку на один уровень с мылом - что я считаю оправданным в связи с тем насколько смазка удобнее в защите флангов и применении.

Commit:
Нерф реактивный брони

Реактивная броня
-Снижена дальность телепорта с 6 до 2.

Реактивная броня всегда являлась одним из самых поломанных хайрисков на станции.Сам факт того что она может поглотить удар/выстрел с 50% шансом уже является головной болью. Прибавить к этому любой щит с 50% шанса блока и на выходе получается кошмарик способный с 75% шансом избежать любого удара или выстрела. И это не беря в расчет предметы и способности антагонистов. Такичая в реактивке и с адреналом до сих пор снится многим в кошмарах. В новой боевке требующей в 4 раза больше попаданий этот предмет стал еще более поломанным.

Commit:
Нерф кожурок от бананогранаты

Эти кожурки определены отдельно и были упущены при изначальной переработке скользких предметов. Данный твик исправляет этот недочет

Jordanstolethepreciousslam commented:
[Changelog.txt](https://github.com/ss220-space/Paradise/files/7920767/Changelog.txt)

Commit:
Упрощение рецепта гарроты

Теперь ввиду нерфа станпрода настало время ввести новую альтернативу - гарроту. Она крафтится почему то из кухонного ножа который хрен достанешь. С этим же коммитом скрафтить гарроту будет так же легко как и станпрод - просто разбить любое стекло, слоать стол в баре и взять провода - вот и готова гаррота.

SergeyPomelov commented:
1. Чейнджлог не полный сильно такое чувство. Нерф мыла нашёл только по коду.
2. Банальное падение на воде теперь сравнимо со смазкой или я пропустил нерф воды?
3. 4-6 выстрелов по движущемуся делать до стаминакрита (это без стимуляторов и защит среди которых куча легкодоступных и легальных)? Это на многом оружии так. Серьёзно? Я что-то тут не понимаю, выглядит как бред. Попробуйте хотя-бы на ровной площадке без мусора, преград и лишних людей попасть столько раз.

Commit:
Ускорение надевания теневых оков в 3 раза.

С тех пор как захват с использованием магии стал довольно длительным процессом, необходимость в таком долгом применении у оков отпала. По сути жертва в стамина крите уже не сможет ничего сделать, а потому длительное применение оков только займет время у нападающего, что прямо не поможет жертве. Жертва получила свой шанс на побег через длительный стан + нападающий получил свой шанс и смог утащить жертву. В длинном периоде заковывания просто нет необходимости.

Commit:
Удвоение дальности "мили" глейра

По результатам тестов тени получились довольно слабыми из-за того что они никогда не страдали столько от станов, сколько они страдали от летала. Тени - один из немногих режимов которые акцентируют внимание на летальном бою. Сами тени убивают медленно но верно тактикой hit-n-run, но сами умирают с пары выстрелов лазером.

Commit:
Финальные твики батона и возвращение станперчаток

Батон
-Стоимость удара увеличена с 400 до 500
-Урон выносливости понижен с 25 до 20
Станпрод
-Урон выносливости увеличен с 20 до 25
Шоковые перчатки
-Добавлен рецепт. Теперь можно сделать из изолей, скотча и проводов. Требует мультитул и кусачки для крафта.
-Длительность оглушения уменьшена с 5 тиков до 1
-Добавлено 20 урона выносливости при попадании.

В связи возросшей сложностью роли антагониста(из-за того факта что теперь жертвы могут легко сбегать), было решено добавить альтернативу станпроду, которая бы была эффективнее в бою основанно на разоружении. Самому станпроду был увеличен стамина урон дабы он оставался альтернативой а не бесполезным куском который не использует никто из-за перчаток.

Jordanstolethepreciousslam commented:
[Changelog.txt](https://github.com/ss220-space/Paradise/files/8010257/Changelog.txt)

Kirozet commented:
проблема: вампир при выбитой стамине восстанавливается одной кнопкой за 20 крови и вгоняет в пермастанкрит другой кнопкой за 0 крови. Поймать вампиров с каким-то опытом игры и прожимания кнопок теперь попросту невозможно даже отрядом в два-три человека. Если он хоть немножко крови набрал, то его уже не поймать. КД глейра, учитывая что он не требует ресурсов, слишком низкое для такого бонуса как станкрит через 5-6 секунд.

Kupelan commented:
Я, наверное, опоздал с предложениями, но не лучше бы было усложнить всё в сторону реализма, а не усложнить механику игры, сделать из тазера аналог реальному с 1-2 зарядами, для станции, ну пусть с тремя максимум, ведь попасть тоже нужно уметь, батонам оставить урон стамине и микростан, тут идея хорошая, в этом есть логика. Фактически у нового и уже более-менее опытного игрока и так под контролем много кнопок и всяческих хоткеев ко всему, но тут попробуй среагируй на каждое противодействие игрока-антагониста, при условии, что в интересах обоих надавать друг другу по щам, а на антагонисте всегда, как правило играют поопытнее. В общем, тазеру надо вернуть тазер, я считаю, иначе логика отдела СБ немного улетучивается вместе с понятным геймплеем. Просто игра на СБ или антаге превратиться в потную катку после которой хочется отдохнуть.
